custom/icds_reports/utils/aggregation_helpers/distributed/ccs_record_monthly.py

```diff
@@ -61,10 +62,13 @@ def aggregation_query(self):
         ).format(end_month_string, start_month_string)
 
         alive_in_month = "(case_list.date_death is null OR case_list.date_death-{}>0)".format(start_month_string)
-        migration_status = "(agg_migration.is_migrated=1 AND agg_migration.migration_date < {})::integer".format(
-            start_month_string)
-        seeking_services = "(person_cases.registered_status IS DISTINCT FROM 0 AND {} IS DISTINCT FROM 1)".format(
-            migration_status)
+        migration_status = "(agg_migration.is_migrated = 1 AND agg_migration.migration_date::date < {" \
```

Comment:
breaking the line partway through a string format like this is really hard to read. despite being longer than the recommended limit, i find the previous formatting much easer to read and understand. alternatively, if you want short lines, breaking it around the `AND` is likely to make it easier to read since each line would be a complete comparison. 
You can also use parentheses or multiline strings as used elsewhere in this file to avoid the `\` which makes it look a bit cleaner. 
These comments apply to all the broken strings

Reply:
updated!